provisioning/roles/drupalvm.vagrant-nfs-fix/tasks/main.yml

```diff
@@ -0,0 +1,32 @@
+---
+- name: Copy Vagrant NFS fix script into place.
+  template:
+    src: vagrant-nfs-fix.j2
+    dest: /usr/sbin/vagrant-nfs-fix
+    owner: root
+    group: root
+    mode: 0755
+
+- name: Copy Vagrant NFS fix init script into place.
+  template:
+    src: vagrant-nfs-fix.init.j2
+    dest: /etc/init.d/vagrant-nfs-fix
+    owner: root
+    group: root
+    mode: 0755
+  when: ansible_service_mgr != 'systemd'
```

Comment:
I believe all supported OSes now include systemd, so we can drop the init script... right?